src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/test_containerapp_commands.py

```diff
@@ -1294,6 +1294,32 @@ def test_containerapp_private_registry_port(self, resource_group):
             JMESPathCheck("properties.configuration.secrets[0].name", f"{acr}azurecrio-{acr}")
         ])
 
+    @AllowLargeResponse(8192)
+    @ResourceGroupPreparer(location="westeurope")
+    def test_containerapp_private_registry_with_username_as_email(self, resource_group):
```

Comment:
For the test, you should include the recording file that created with command `azdev test test_containerapp_private_registry_with_username_as_email --live`
The pipeline run with the recording file in the playback mode.

Reply:
Hi @Greedygre @yanzhudd I've removed the E2E test since ACR/MCR does not support email addresses as a username input. However, supporting email addresses as a username is an important requirement for one of our major customers. Given this limitation, I'm requesting an exception for this E2E test.

Reply:
@Greedygre correct. Customer are wanting to use azure container apps with JFrog, Docker, etc. Registries that allow a email address as login: Yes, I've updated the description with the screenshot.